microprofile/WFLY-10522_microprofile-config-smallrye_extension.adoc

```diff
@@ -0,0 +1,58 @@
+= [WFLY-10522] Support Eclipse MicroProfile Config
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:keywords:          microprofile,config,openshift
+
+== Overview
+
+Support Eclipse MicroProfile Config for applications deployed in WildFly.
+The integration uses the smallrye-config component to provide the MicroProfile Config implementation.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-10522[WFLY-10522]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1016[EAP7-1016]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
```

Comment:
Please add me and @mnovak1 as a QE Contacts

Reply:
done

Reply:
There has been a little change. Can you replace the standing contacts by me?